appinfo.json

```diff
@@ -6,6 +6,8 @@
 	"id": "com.palm.launcher",
 	"icon": "icon.png",
 	"theme":"palm-dark",
-	"visible": "false",
+    "defaultWindowType": "system_ui",
```

Comment:
Spacing on these 2 lines seems off compared to others?

Reply:
Ah, tabs...